ocp_install_env.sh

```diff
@@ -133,7 +133,7 @@ EOF
 
 function additional_trust_bundle() {
   if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ]; then
-    if [ -z $MIRROR_IMAGES ]; then
+    if [ -z "${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES:-}}" ]; then
```

Comment:
`ENABLE_LOCAL_REGISTRY` is already defined also in terms of `MIRROR_IMAGES`. In case this can be reviewed https://github.com/openshift-metal3/dev-scripts/blob/5b8ec2c49578f4a47338b81471277664501665de/02_configure_host.sh#L55 and CBO check extracted

Reply:
What do you mean by extracting CBO?, extracting from 
`export ENABLE_LOCAL_REGISTRY=${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES:-${ENABLE_CBO_TEST:-$(env | grep "_LOCAL_IMAGE=")}}} ` or extracting at additional_trust_bundle function?

Reply:
Because if we extract CBO from here `export ENABLE_LOCAL_REGISTRY=${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES:-${ENABLE_CBO_TEST:-$(env | grep "_LOCAL_IMAGE=")}}} `, we need to add CBO_TEST into https://github.com/openshift-metal3/dev-scripts/blob/5b8ec2c49578f4a47338b81471277664501665de/02_configure_host.sh#L57

Reply:
Sorry, meant to split the current definition of `ENABLE_LOCAL_REGISTRY` and keep it just as `export ENABLE_LOCAL_REGISTRY=${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES}}`, in the assumption that MIRROR_IMAGES indicates a super-feature of ENABLE_LOCAL_REGISTRY